bigtable-hbase-1.x-parent/bigtable-hbase-1.x-hadoop/pom.xml

```diff
@@ -30,12 +30,18 @@ limitations under the License.
     bigtable-hbase-1.x.
   </description>
 
+  <properties>
+    <!-- define a property that can be ignored by renovate -->
+    <hbase1-hadoop-slf4j.version>1.6.1</hbase1-hadoop-slf4j.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>bigtable-hbase-1.x-shaded</artifactId>
       <version>${project.version}</version>
       <exclusions>
+        <!-- hbase-shaded-client will be replaced with hbase-client -->
```

Comment:
nit: add a TODO infront of this comment

Reply:
This just explaining whats happening in this pom: a few lines below we are adding hbase-client which was removed on this line